### PR TITLE
Add list of metrics reported by NOM

### PR DESCRIPTION
**Pull request by alexnb**
[Trello card](https://trello.com/c/fxLl0DFk/2211-create-a-list-of-metrics-exposed-by-nom)

----

If you open a PR that needs to go into a current version, you need to *cherry-pick your commit from dev over to the current version branch*. Only then will the proper builds that generate html/pdf be run. But beware: Docs will be generated but not published automatically!

- [x] N/A - or - I have added the appropriate "cherry-pick-to" labels to this PR so I don't forget to do this later!

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

File: modules/ROOT/pages/monitoring/metric-manager.adoc [hunk 1]
```diff
@@ -17,6 +17,57 @@ Charts allow for zooming/panning by clicking and specifying the area of interest
...
+** Active Write Transactions
+** Committed Transactions
+** Last Transaction ID
+** Peak Concurrent Transactions
+** Transaction Rollbacks
+** Allocated IDs (Properties)
+** Allocated IDs (Relationships)
+** Allocated IDs (Nodes)
+** Replan Events
+** Disk Used (/Data)
```
Comment (ravikiran-sulikeri) [inline modules/ROOT/pages/monitoring/metric-manager.adoc:58]: `Disk Used` is repeated, may be add `(per database)`?

Comment (alexnb) [reply modules/ROOT/pages/monitoring/metric-manager.adoc:58]: Actually it is under the `Database metrics (per Neo4j database)` category. Do you think we still need to add "per database"?

Comment (ravikiran-sulikeri) [reply modules/ROOT/pages/monitoring/metric-manager.adoc:58]: ah didn't see the heading, no I guess that should be enough.

**Review by ravikiran-sulikeri — APPROVED** [review]
LGTM 👍

**Review by AlexicaWright — COMMENTED** [review]
Why are the metrics all in Camel Case?

**Comment by alexnb** [comment]
> Why are the metrics all in Camel Case?

They are names so in the UI, see below
image

**Review by SeBBBe — APPROVED** [review]

**Pull request merged**